### PR TITLE
feat: add working-directory option, new-release-published output

### PR DESCRIPTION
**Pull request by benjyiw**
- feat: implement working-directory option

- fix: add a flag for checking out the repository

- feat: add new-release-published output

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -27,28 +27,41 @@ inputs:
   delete-branch:
     description: 'Delete PR branch after merge'
     required: false
-    default: true
+    default: 'true'
   branch-prefix:
     description: 'The prefix for the name of the feature branch'
     required: false
     default: 'automated-code-release-'
+  working-directory:
```
Comment (codelite7) [inline action.yaml:35]: Since this is being used, does this not need a default?

File: action.yaml [hunk 2]
```diff
@@ -27,28 +27,41 @@ inputs:
...
+    description: 'Directory to run semantic-release in '
+    required: false
```
Comment (todpunk) [inline action.yaml:37]: Uh... not required, but no default?

File: action.yaml (hunk 1 above)
Comment (benjyiw) [reply action.yaml:35]: it doesn't need a default. if you don't specify a value, then it's like the option is never specified to the nested action, so the default is whatever the nested actions default is. the cycjimmy/semantic-release-action doesn't have a default for me to copy explicitly, so I just left it blank

**Review by codelite7 — APPROVED** [review]

File: action.yaml (hunk 2 above)
Comment (benjyiw) [reply action.yaml:37]: I believe that no default is essentially null or undefined, so it's like the parameter that uses the input is never actually specified.

**Pull request merged**